lightwood/encoders/text/vocab.py

```diff
@@ -0,0 +1,34 @@
+import torch
+import flair
+from transformers import DistilBertTokenizer
+
+from lightwood.encoders.encoder_base import BaseEncoder
+
+
+class VocabularyEncoder(BaseEncoder):
+    def __init__(self, is_target=False):
+        super().__init__(is_target)
+        self._tokenizer_class = DistilBertTokenizer
+        self._pretrained_model_name = 'distilbert-base-uncased'
+        self._max_len = None
```

Comment:
To avoid searching for member variables definitions they can be initialized to None in `__init__()`:
```
# These are initialized in self.prepare_encoder()
self._max_len = None
self._tokenizer = None
self._pad_id = None
```

Reply:
Solved